tests/lib/rules/scoped-elements.js

```diff
@@ -132,14 +132,38 @@ class MyElement extends ScopedElementsMixin(LitElement) {
     // ❌ INVALID
     //------------------------------------------------------------------------------
     invalid: [
+      {
+        /** ❌ Not declared */
+        code: `
+class MyElement extends ScopedElementsMixin(LitElement) {
+render() { return html\`<my-el></my-el>\`}
+}
+    `,
+        errors: [{ message: 'Element not declared in static scopedElements' }]
+      },
+      {
+        /** ❌ Not declared */
+        code: `
+class MyElement extends ScopedElementsMixin(LitElement) {
+connectedCallback() {
+  super.connectedCallback();
```

Comment:
Currently in your code, if there is ANY kind of `super.whatever()` call present, it will ignore linting scoped elements. So also if you use `super.connectedCallback()`, it'll skip linting scoped elements. Added a test to catch this 🙂